label.js

```diff
@@ -19,7 +19,20 @@ async function label() {
   const context = github.context;
   const repoName = context.payload.repository.name;
   const ownerName = context.payload.repository.owner.login;
-  const issueNumber = context.payload.issue.number;
+  var issueNumber;
+
+  if (context.payload.issue !== undefined) {
+    issueNumber = context.payload.issue.number;
+  } else if (context.payload.pull_request !== undefined) {
+    issueNumber = context.payload.pull_request.number;
+  } else if (context.payload.project_card !== undefined) {
+    var issuesUrl = context.payload.project_card.issues_url;
+    var urlArray = issuesUrl.split("/");
+    issueNumber = urlArray[urlArray.length - 1];
```

Comment:
```suggestion
    issueNumber = context.payload.project_card.issues_url.split("/").pop()
```

Although shorter may not always be clearer 😄

Reply:
I don't mind. I'm not a JS developer :rofl: